frontend/src/metabase/dashboard/components/ParametersPopover.tsx

```diff
@@ -37,32 +47,42 @@ export const ParametersPopover = ({ onClose, onAddParameter }) => {
           const defaultOption =
             defaultOptionForParameterSection[parameterSection.id];
 
-          onAddParameter(defaultOption);
+          if (defaultOption) {
+            onAddParameter(defaultOption);
+          }
           onClose();
         }
       }}
     />
   );
 };
 
-const ParameterOptionsSection = ({ section, onClick }) => (
+const ParameterOptionsSection = ({
+  section,
+  onClick,
+}: {
+  section: ParameterSection;
+  onClick: () => void;
+}) => (
   <OptionItemRoot onClick={onClick}>
     <OptionItemTitle
       className={cx(CS.textBold, CS.flex, CS.alignCenter)}
       style={{ marginBottom: 4 }}
     >
-      <Icon
-        size="16"
-        name={getParameterIconName(section.id)}
-        className={CS.mr1}
-      />
+      <Icon size="16" name="label" className={CS.mr1} />
```

Comment:
typescript showed that `getParameterIconName` expected parameter object and not id, so it never worked as intended and nobody noticed. I just hardcoded the fallback value from `getParameterIconName`